packages/client-core/src/components/UserMediaWindow/index.tsx

```diff
@@ -434,6 +435,13 @@ export const UserMediaWindow = ({ peerID, type }: Props): JSX.Element => {
     document.getElementById(peerID + '-' + type + '-audio-container')!.append(audioElement)
   }, [])
 
+  useEffect(() => {
+    if (!videoStream) return
+    const mediaNetwork = Engine.instance.mediaNetwork as SocketWebRTCClientNetwork
+    if (isPiP) setPreferredConsumerLayer(mediaNetwork, videoStream as ConsumerExtension, 3)
```

Comment:
Are these layer numbers belonging to an enum? If so, we should use them directly, and if not, we should define one rather than using magic numbers.

Reply:
I changed things some after seeing this comment. I was actually not using the lowest layer available; the layers are 0-indexed, and the defaults have been changed to reflect this. Instead of hard-coding max to '3', it's now checking how many layers there are from scalabilityMode, and then making some calculations from there.